### PR TITLE
[#2151] Update LoadingOverlay and Minor Versions of Node Dependencies

### PR DESCRIPTION
**Pull request by sopa301**
Part of #2151.

### Proposed commit message

```
Update LoadingOverlay and Minor Versions of Node Dependencies

Some dependencies are not at their latest minor or patch releases.

Let's update these dependencies, as well as LoadingOverlay as part of a
bug fix.
```

### Other information

There was a bug that arose when updating Vue to `3.3` onwards. I went ahead and updated `vue-loading-overlay` while fixing it.

Cypress has a latest version `13.6.6`, but it was recommended to keep to `13.6.4` due to a [bug](https://github.com/cypress-io/cypress/issues/29085).

### CODE REVIEW

**Review by asdfghjkxd — APPROVED** [review]
LGTM!

**Review by MarcusTXK — APPROVED** [review]
LGTM

**Review by ckcherry23 — COMMENTED** [review]
Thanks for upgrading the dependencies @sopa301! Please also resolve the conflicts.

File: frontend/src/app.vue [hunk 1]
```diff
@@ -36,7 +36,7 @@ const loadingResourcesMessage = 'Loading resources...';
 const app = defineComponent({
   name: 'app',
   components: {
-    LoadingOverlay,
+    Loading,
```
Comment (ckcherry23) [inline frontend/src/app.vue:39]: Can this still be named `LoadingOverlay`? I think that would make it more meaningful.

**Review by ckcherry23 — APPROVED** [review]
LGTM!

**Pull request merged**